Sources/Kitura/bodyParser/BodyParser.swift

```diff
@@ -215,12 +215,15 @@ public class BodyParser: RouterMiddleware {
                     // end boundary found, end of parsing
                     return .multipart(parts)
                 }
-                state = .body
+                state = .header
+            }
+
```

Comment:
If the line has boundary data it should not fall to the switch, instead we should add a `continue`.